### PR TITLE
Fix/solar 270/allow section visibility filter apply on get navigation elements by group

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
This change will allow disabling all navigation buttons using the feature flag with SectionVisibilityFilter config. 

How to use: 
- Create migration that will add desired section (perspective) to be disabled base on feature flag
- Define feature flag
- Clear cache

### CODE REVIEW

File: actions/class.Main.php [hunk 1]
```diff
@@ -401,7 +401,22 @@ public function index()
     private function getNavigationElementsByGroup($groupId)
     {
         $entries = [];
+
+        /**
+         * @var int $i
+         * @var  Perspective $perspective
+         */
         foreach (MenuService::getPerspectivesByGroup($groupId) as $i => $perspective) {
```
Comment (Karol-Stelmaczonek) [inline actions/class.Main.php:409]: variable $i is never used

File: actions/class.Main.php [hunk 2]
```diff
@@ -401,7 +401,22 @@ public function index()
...
+            if (!$this->getSectionVisibilityFilter()->isVisible($perspective->getId())) {
```
Comment (Karol-Stelmaczonek) [inline actions/class.Main.php:410]: $perspective could be a null so will result in fatal if method is called on null
```suggestion
            if ($perspective && !$this->getSectionVisibilityFilter()->isVisible($perspective->getId())) {
```

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: actions/class.Main.php (hunk 1 above)
Comment (bartlomiejmarszal) [reply actions/class.Main.php:409]: 🤷‍♂️

**Pull request merged**